lib/esp-scanning-lib.robot

```diff
@@ -0,0 +1,173 @@
+*** Settings ***
+Documentation       Collection of keywords related to UEFI Secure Boot
+
+Resource            ../keywords.robot
+
+
+*** Variables ***
+${SHIMX64_URL}=     https://cloud.3mdeb.com/index.php/s/TipPHgsCj6MCZHi/download/shimx64.efi
+${GRUBX64_URL}=     https://cloud.3mdeb.com/index.php/s/e2JjYANjMsgyXtS/download/grubx64.efi
+${TINYCORE_URL}=    https://distro.ibiblio.org/tinycorelinux/14.x/x86/release/CorePlus-current.iso
+${DTS_URL}=         https://dl.3mdeb.com/open-source-firmware/DTS/v1.2.8/dts-base-image-v1.2.8.iso
+
+
+*** Keywords ***
+Prepare EFI Partition With System Files
+    [Documentation]    Prepares a bootable EFI partition with OS files
+    ...    by mounting a hard drive in Qemu, or creating files on the DUT
+    ...    via PiKVM
+
+    IF    "${MANUFACTURER}" == "QEMU"
+        Add HDD To Qemu    img_name=image.img
+    ELSE
+        IF    "${DUT_CONNECTION_METHOD}" == "pikvm"
+            Boot System Or From Connected Disk    ubuntu
+            Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
```

Comment:
Why do you need to set this? The `Login to Linux` already sets the DUT_CONNECTION_METHOD to SSH when pikvm is used. This feels redundant.

Reply:
Removed in [commit a90d5e0](https://github.com/Dasharo/open-source-firmware-validation/commit/a90d5e01100174c8b883b01ae914d4693f595747#diff-1c553d8d90529e6578e4c3b7bec1aea478c26edce6d99d353b8a35b02c05c67cR26)